sisyphus/localengine.py

```diff
@@ -341,3 +342,9 @@ def get_task_id(self, task_id):
             "Job in local engine started without task_id, worker is probably started manualy. Continue with task_id=1"
         )
         return 1
+
+    def get_job_node_hostnames(self) -> List[str]:
+        """
+        :return: we are running on `["localhost"]` since this is a local engine.
+        """
+        return ["localhost"]
```

Comment:
is the string "localhost" enough for your purposes or should we try to determine the hostname of the local host?